data/json/items/generic.json

```diff
@@ -2361,6 +2361,47 @@
     "price": 0,
     "price_postapoc": 0
   },
+  {
+    "type": "GENERIC",
+    "id": "bouquet",
+    "name": { "str_sp": "flower bouquet" },
+    "description": "A collection of flowers elegantly arranged in a glass vase.",
```

Comment:
Do we really need a glass vase part here?

Reply:
So just 'a collection of flowers elegantly arranged in a vase?' Or 'a collection of elegantly arranged flowers?' Glass makes up part of the materials of the bouquet currently.

Reply:
I was also conflicted about whether to make the flowers plastic to begin with, since I imagine depending on if you find them on day 1 of the apocalypse or day 1001, the flowers will look very different. Whereas, if it was plastic to begin with, then it wouldn't matter.

Reply:
You can call it something like "withered bouquet" and call it a day.